beetmoverscript/templates/fennec_candidates.yml

```diff
@@ -127,13 +127,13 @@ mapping:
       destinations:
         - SOURCE.asc
     source.tar.xz:
-      s3_key: source.tar.xz
+      s3_key: source/fennec-{{ version}}.source.tar.xz
```

Comment:
I don't think we even offered this before for Fennec, so it's a nice win along the way (e.g. https://archive.mozilla.org/pub/mobile/candidates/58.0b3-candidates/build4/)